pkg/controller/datavolume-controller.go

```diff
@@ -43,19 +43,24 @@ import (
 	cdischeme "kubevirt.io/containerized-data-importer/pkg/client/clientset/versioned/scheme"
 	informers "kubevirt.io/containerized-data-importer/pkg/client/informers/externalversions/datavolumecontroller/v1alpha1"
 	listers "kubevirt.io/containerized-data-importer/pkg/client/listers/datavolumecontroller/v1alpha1"
-	. "kubevirt.io/containerized-data-importer/pkg/common"
+	"kubevirt.io/containerized-data-importer/pkg/common"
```

Comment:
In clone-controller.go, we import as "cdi".
Now we have both :
glog.V(common.Vdebug)
glog.V(cdi.Vdebug)

Reply:
Ahh, yeah thanks for noticing that; sorry about that; I'll update this in the morning and make it consistent and just use "common".  Sound ok?

Reply:
Great! Thanks.